orderbook/openapi.yml

```diff
@@ -417,6 +417,49 @@ paths:
                   $ref: "#/components/schemas/Order"
         400:
           description: Problem with parameters like limit being too large.
+  /api/v1/quote:
+    post:
+      summary: Quotes a price and fee for the specified order parameters.
+      description: |
+        This API endpoint accepts a partial order and computes the minimum fee and
+        a price estimate for the order. It returns a full order that can be used
+        directly for signing, and with an included signature, passed directly to
+        the order creation endpoint.
+      parameters:
```

Comment:
I find a bit strange to mix query params and body info. I would stick to `requestBody` and add the additional info as optional fields there

Reply:
This was a header parameter. It has been removed though.

Reply:
My bad, i thought it was a parameter 👌